internal/services/springcloud/spring_cloud_container_deployment_resource.go

```diff
@@ -79,6 +81,16 @@ func resourceSpringCloudContainerDeployment() *pluginsdk.Resource {
 				DiffSuppressFunc: pluginsdk.SuppressJsonDiff,
 			},
 
+			"application_performance_monitoring_resource_ids": {
```

Comment:
When we're referencing the ids of other resources in the azurerm provider we don't specify resource. This should be renamed to 
```suggestion
			"application_performance_monitoring_ids": {
```

Reply:
Got it!